Gems/Atom/RPI/Code/Source/RPI.Private/PerformanceCVarManager.cpp

```diff
@@ -0,0 +1,122 @@
+
+#include "PerformanceCVarManager.h"
+
+namespace AZ
+{
+    namespace RPI
+    {
+
+        AZ::Debug::PerformanceCollector::DataLogType GetDataLogTypeFromCVar(const AZ::CVarFixedString& newCaptureType)
+        {
+            if (newCaptureType[0] == 'a' || newCaptureType[0] == 'A')
+            {
+                return AZ::Debug::PerformanceCollector::DataLogType::LogAllSamples;
+            }
+            return AZ::Debug::PerformanceCollector::DataLogType::LogStatistics;
+        }
+
+        class PerformanceCvarManager
+        {
+        public:
+            static constexpr char LogName[] = "RPIPerformanceCvarManager";
+
+            static void OnDataLogTypeChanged(const AZ::CVarFixedString& newCaptureType)
+            {
+                AZ_TracePrintf(LogName, "%s cvar changed to %s", __FUNCTION__, newCaptureType.c_str());
+                auto performanceCollectorOwner = PerformanceCollectorOwner::Get();
+                if (!performanceCollectorOwner)
+                {
+                    return;
+                }
+                auto performanceCollector = performanceCollectorOwner->GetPerformanceCollector();
+                if (!performanceCollector)
+                {
+                    return;
+                }
+                performanceCollector->UpdateDataLogType(GetDataLogTypeFromCVar(newCaptureType));
+            }
+
+            static void OnNumberOfCaptureBatchesChanged(const AZ::u32& newValue)
+            {
+                AZ_TracePrintf(LogName, "%s cvar changed to %u", __FUNCTION__, newValue);
+                auto performanceCollectorOwner = PerformanceCollectorOwner::Get();
+                if (!performanceCollectorOwner)
+                {
+                    return;
+                }
+                auto performanceCollector = performanceCollectorOwner->GetPerformanceCollector();
+                if (!performanceCollector)
+                {
+                    return;
+                }
+                performanceCollector->UpdateNumberOfCaptureBatches(newValue);
+            }
+
+            static void OnWaitTimePerCaptureBatchChanged(const AZ::u32& newValue)
+            {
+                AZ_TracePrintf(LogName, "%s cvar changed to %u", __FUNCTION__, newValue);
+                auto performanceCollectorOwner = PerformanceCollectorOwner::Get();
+                if (!performanceCollectorOwner)
+                {
+                    return;
+                }
+                auto performanceCollector = performanceCollectorOwner->GetPerformanceCollector();
+                if (!performanceCollector)
+                {
+                    return;
+                }
+                performanceCollector->UpdateWaitTimeBeforeEachBatch(AZStd::chrono::seconds(newValue));
+            }
+
+            static void OnFrameCountPerCaptureBatchChanged(const AZ::u32& newValue)
+            {
+                AZ_TracePrintf(LogName, "%s cvar changed to %u", __FUNCTION__, newValue);
```

Comment:
A newline needs to be added to the `AZ_TracePrintf`
```suggestion
                AZ_TracePrintf(LogName, "%s cvar changed to %u\n", __FUNCTION__, newValue);
```

Reply:
Fixed.